src/Util/Cloner.php

```diff
@@ -17,7 +17,7 @@
 final class Cloner
 {
     /**
-     * @psalm-template OriginalType
+     * @psalm-template OriginalType of object
```

Comment:
since the native type is already `object` this template narrowing is not a bc break, but just a bugfix to make PHPStan happier